### PR TITLE
style burger menu, position of true/false answers in desktop view, si…

### PR DESCRIPTION
**Pull request by FrancoisSalmon**
…ze of question title, rework of media query for rules page, style of team page

### CODE REVIEW

**Review by remi59800 — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/components/Archi/Burger/style.jsx [hunk 1]
```diff
@@ -21,24 +21,26 @@ export default styled.div`
...
     background-color: var(--main-color);
     opacity: 0.9;
+    border-radius: 10px;
   }
 
   a {
     text-decoration: none;
     color: var(--white-color);
-    font-size: 1.5rem;
+    font-size: 1.08rem;
```
Comment (LoicBrassart) [inline frontend/src/components/Archi/Burger/style.jsx:37]: That's precise !

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**